config/locales/models/report.en.yml

```diff
@@ -51,16 +51,6 @@ en:
           <p class="govuk-body">
             For in-depth guidance, refer to <a href="https://beisodahelp.zendesk.com/hc/en-gb/search?utf8=%E2%9C%93&query=uploading+new+activities">uploading new activities</a> and <a href="https://beisodahelp.zendesk.com/hc/en-gb/search?utf8=%E2%9C%93&query=update+activities">uploading updates to activities</a> in the help centre.
           </p>
-      transactions:
-        heading: Actuals in this report
-        per_activity_heading: Actual spend per activity
-        guidance_html:
-          <p class="govuk-body">
-            This page shows all the actual spend you have reported in RODA since the last reporting quarter.
-          </p>
-          <p class="govuk-body">
-            For in-depth guidance, refer to <a href="https://beisodahelp.zendesk.com/hc/en-gb/search?utf8=%E2%9C%93&query=uploading+new+actuals">uploading new actuals</a> and <a href="https://beisodahelp.zendesk.com/hc/en-gb/search?utf8=%E2%9C%93&query=update+activities">uploading updates to activities</a> in the help centre.
-          </p>
```

Comment:
If we no longer believe that using translations is a good idea, should we document that decision to more away from the convention in an Architecture Decision Record?

Personally I tend to think it's a good practice as it:

- keeps the views smaller
- avoids involving the view in a commit which only changes copy
- often allows terminology to be changed in one place (e.g. all occurrences of "Financial" can be changed to "Actual" in one edit, perhaps!)
- allows copy to be managed in external tooling which synchronises with translation files (e.g. https://phrase.com)
- arguable encourages a more structured approach to designing content

But I know it can be drag following the additional "piece of indirection" to locate a translation or find where a particular translation is used. I'd be interested to know how others feel -- and what other teams are doing...

Reply:
One of my favourite topics! 😄 

I also would like to get a feel for this one from other folks! For me, it is a tough balance to strike for sure.

For me personally, the cons of the locales outweigh the pros on a project like this, and, of course, each project brings its own context.

I definately agree that an ADR at the outset would set the expectation to follow and keep us all aligned - this is a good thing.

To your points I would counter:

- having the content in the view maps to some folks mental model (view = content not config = content)
- I not sure I have seen many locales that are robust enough to support only changing the locales file (although I probably have seen some!)
- In principle I agree that a change like "Transaction" to "Actual spend" should be possible, but again I have yet to work on a codebase where that level of commitment to the locales has been followed (but I admint I have not worked on many!)
- I've never actually had to localise a project, I appreciate the view that having everything in the locales would make this much easier, but in a lot of cases it feel like cart before horse to me, for example, this project has no scope to localise to a langauge other than English

I would add to my list of negatives:

- I've worked on two projects where we had a content designer who was asked to update the content in github and the locales generally made this harder for them (they generally work with what I would consider views)
- related to above, the content without context is unhelpful for some (I would say particularly folks show are newer to the concept)
- with no formal struture to the locales, they become a hugh mess over time, with different keys for the same content (I was tasked with attempting to sort out this projects some time ago, follow that story here: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/446)
- when is a new key required and how that key is expressed (named) seems more art than science
- how the entire locales files are organised also seems to be a matter of preference for indivuals with no 'linter' to take the decision away for us (Again, I would welcome some formalised approch in an ADR if such a thing exists!)
- I find tests with locales harder to parse generally

and there are loads of positives too, I like:

- not having to update the tests when content changes (but at the cost of the tests being less readable)

All that said, you are right to call me out on this - we do rely on the locales on this project and I need to strive for consistency regardless of my own feeliings.

Reply:
All your contra-arguments are valid. 

I'm very happy to go with the in-line approach if that's generally preferred by the team. We can document our decision and gradually move away from "translations"...